vespajlib/src/test/java/com/yahoo/geo/OneDegreeParserTestCase.java

```diff
@@ -0,0 +1,266 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.geo;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests for the OneDegreeParser class.
+ *
+ * @author arnej27959
+ */
+public class OneDegreeParserTestCase {
+
+    private static final double delta = 0.000000000001;
+
+    private OneDegreeParser parser;
+
+    private void checkLat(boolean ans, String to_parse, double expected) {
```

Comment:
Prefer camelCase for Java code. Applies for entire file.

Reply:
fixed.